use-timescale/fork-services.md

```diff
@@ -0,0 +1,183 @@
+---
+
+import IntegrationPrereqs from "versionContent/_partials/_integration-prereqs.mdx";
+import CLIFORKS from "versionContent/_partials/_devops-cli-service-forks.mdx";
+
+# Developer $SERVICE_SHORT forks
+
+Modern development is highly iterative. Developers and AI agents need safe spaces to test changes before deploying them
+to production. Forkable $SERVICE_SHORTs make this natural and easy. Spin up a branch, run your test, throw it away, or
+merge it back.
+
+A fork is an exact copy of a $SERVICE_SHORT at a specific point in time, with its own independent data and configuration,
+including:
+- The database data and schema
+- Configuration
+- An admin `tsdbadmin` user with a new password
+
+Forks are fully independent. Changes to the fork don't affect the parent $SERVICE_SHORT, you can query
+them, run migrations, add indexes, or test new features.
+
+Forks are a powerful way to share production-scale data safely. BI and data science teams often need access to real
+datasets to build models or generate insights. With forkable $SERVICE_SHORTs, you easily create instant, zero-copy
+branches of a production $SERVICE_SHORT that is isolated from production, but contains all the data needed for
+analysis. You share this fork with your analytics teams in seconds. This dramatically reduces friction getting insights
+from live data. 
+
+## Understand $SERVICE_SHORT forks
+
+You use $SERVICE_SHORT forks for disaster recovery, CI/CD automation, and testing and development. For example, you 
+can automatically test a major $PG upgrade on a fork before applying it to your production $SERVICE_SHORT. 
+
+$CLOUD_LONG offers the following fork strategies:
+
+- `now`: create a fresh fork of your database at the current time.
+   Use when:
+   - You need the absolute latest data
+   - Recent changes must be included in the fork
+
+- `last-snapshot`: fork from the most recent [automatic backup or snapshot][automatic-backups]. 
+  Use when:
+  - You want the fastest possible fork creation
+  - Slightly behind current data is acceptable
+
+- `timestamp`: fork from a specific point in time within your [retention period][pricing].
+  Use when:
+  - Disaster recovery from a known-good state
+  - Investigating issues that occurred at a specific time
+  - Testing "what-if" scenarios from historical data
+
+The retention period for point-in-time recovery and forking depends on your [pricing plan][pricing-plan-features].
+
+### Fork creation speed
+
+Fork creation speed depends on your pricing plan.
+
+- Free plan: ~30-90 seconds. Uses a Copy-on-Write storage architecture with zero-copy between a fork and the parent.
+- Paid plan: varies with the size of your $SERVICE_SHORT, typically 5-20+ minutes. Uses tradional storage architecture
+   with backup restore + WAL replay.
+
+### Billing
+
+You can fork a service created in the free plan to a free or a paid plan. However, you cannot fork a service created in 
+a paid plan to a free plan service. 
```

Comment:
I'm maybe not the best person to ask, but I feel like this is mixing up the free and paid _plans_ with free vs paid _services_.  I believe the "plan" generally refers to the overall billing plan that the project is on, whereas the important thing here is whether the services _themselves_ are free or paid (because even paid plans can create 2 free services).